gk/main.c

```diff
@@ -2299,6 +2299,8 @@ cleanup_gk(struct gk_config *gk_conf)
 	unsigned int ui;
 
 	for (i = 0; i < gk_conf->num_lcores; i++) {
+		gk_conf->instances[i].mp = NULL;
```

Comment:
Instead of copying this comment everywhere, we are better off creating a function to "destroy" the memory pool. The function would be empty, just have the comment inside. But this would allow us to have a proper code everywhere. Thus, if a destroy function is ever included in DPDK, our code would be ready to adopt it.